db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/amazon_check_pre_retirement.rb

```diff
@@ -12,7 +12,7 @@
   $evm.log('info', "Instance:<#{vm.name}> on EMS:<#{ems.try(:name)} has Power State:<#{power_state}>")
 
   # If VM is powered off or this instance is running on an instance store
-  if %w(off suspended).include?(power_state) || vm.hardware.root_device_type == "instance_store"
+  if %w(off suspended terminated).include?(power_state) || vm.hardware.root_device_type == "instance_store"
```

Comment:
@tinaafitz can you have a look here? I'm not so sure what retirement means in our case. Terminated instances will be deleted from ec2 automatically